lib/main.dart

```diff
@@ -42,11 +43,8 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp.router(
-      theme: ThemeData(
-        primarySwatch: Colors.blue,
-        brightness: Brightness.light,
-        fontFamily: Assets.fonts.neuzeit,
-      ),
+      theme: AppTheme.light
+          .copyWith(scaffoldBackgroundColor: const Color(0x00000000)),
```

Comment:
What do you think about using `Colors.transparent` here? 🙏

Reply:
Resolved in https://github.com/nimblehq/ic-flutter-avishek/pull/31/commits/1d21b6a8cbb3231ed9ae20adcca1c90f6b0eb235